php-frontend/src/test/java/org/sonar/php/cfg/ControlFlowGraphTest.java

```diff
@@ -46,133 +45,128 @@
  * -- 'pred' is a bracketed array of expected predecessor ids
  *
  * 2. each basic block must contain the 'block' function call as the first statement
```

Comment:
what is basic block?

Reply:
This is the standard terminology for a code block that executes sequentially - https://en.wikipedia.org/wiki/Basic_block